lib/omniauth/strategies/shopify.rb

```diff
@@ -49,7 +49,7 @@ def valid_signature?
       def valid_scope?(token)
         params = options.authorize_params.merge(options_for("authorize"))
         return false unless token && params[:scope] && token['scope']
-        (params[:scope].split(',').sort == token['scope'].split(',').sort)
+        (params[:scope].delete(' ').split(',').sort == token['scope'].delete(' ').split(',').sort)
```

Comment:
By default, `params[:scope]` have spaces in them in a number of Shopify API gems, :ship: this PR ASAP IMO.

@EiNSTeiN- This is the issue I mentioned to you.